gopkgs.go

```diff
@@ -384,9 +384,6 @@ func listMods(workDir string) ([]mod, error) {
 	for s.Scan() {
 		line := s.Text()
 		ls := strings.Split(line, ";")
-		if err != nil {
```

Comment:
@uudashr I've removed this check, seems like a leftover from 616f72fc9386c350371ad872d5fcb2b01d7aa31e